crates/astria-sequencer/src/app.rs

```diff
@@ -392,20 +392,23 @@ impl App {
         transaction::execute(&signed_tx, &mut state_tx)
             .await
             .context("failed executing transaction")?;
-        state_tx.apply();
+        let (_, events) = state_tx.apply();
 
         // note: deliver_tx is now called (internally) before begin_block,
         // so increment the logged height by 1.
         let height = self.state.get_block_height().await.expect(
             "block height must be set, as `begin_block` is always called before `deliver_tx`",
         );
         info!(
+            tx_hash = hex::encode(signed_tx.hash()),
             ?signed_tx,
```

Comment:
If we touch this event let's fix it to remove this field entirely. It's too verbose.